tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "test", "build"]
+  "exclude": ["node_modules", "__tests__", "doc", "build"]
```

Comment:
Should we add some pattern for ignoring unit tests too? Something like `*/**/*.test.ts`.
The coverage folder must be ignored too.

Reply:
Done @Bounteous17 !